src/doc/book/structs.md

```diff
@@ -61,7 +61,7 @@ write something like this:
 
 ```rust,ignore
 struct Point {
-    mut x: i32,
+    mut x: i32, // this causes an error
```

Comment:
Copied this comment (capitalization, punctuation, spacing) from line 87 of the same file for consistency. The code sample comments in this file (lines 85, 87, and 237-238) are all lower case and without periods (terminating ellipsis at line 237). Formatting is similar in other files, e.g. "mutability.md", "traits.md". Let me know.